install/kubernetes/helm/istio/values.yaml

```diff
@@ -150,11 +151,25 @@ global:
 
   # If set to true, the pilot and citadel mtls will be exposed on the
   # ingress gateway
-  meshExpansion: false
-
-  # If set to true, the pilot and citadel mtls and the plain text pilot ports
-  # will be exposed on an internal gateway
-  meshExpansionILB: false
+  meshExpansion:
+    enabled: false
+    # If set to true, the pilot and citadel mtls and the plain text pilot ports
+    # will be exposed on an internal gateway
+    useILB: false
+
+  multiCluster:
```

Comment:
Per my general comment I think you need a different moniker to represent this mode.  Using "multicluster" will cause confusion with the existing 1.0 functionality

Reply:
Sorry I do not want to confuse end users by creating more and more monikers for the same thing. Multicloud means several things to different people. As such, I don't want to use such marketing terms in the system either. This is a system to connect multiple k8s clusters in same or different AZs, isn't it? Does it matter if the underlying technology is based on a VPN+flat-network or a non-VPN gateway based tunneling mode/ZeroVPN ?

Reply:
I don't have the benefit of seeing the documentation you plan to produce along with this PR,  but this isn't about marketing terms.  For better or worse we have chosen to use the term "multicluster" to describe a deployment that has a single pilot managing multiple clusters.  This change is primarily deployment code and assumes a single pilot manages a single cluster.   Calling 2 different deployment models by the same name is confusing.  I don't see how you could see it any other way.   

It would be much better to chose to describe this moniker using a different naming.

Reply:
There is nothing in this pr to suggest two pilot mode. You could use this with the single pilot for all clusters. This is simply about using the gateway for communication over tls instead of assuming pod network spanning.